frontend/src/components/dashboard/page_timings/PageTimingsTable.vue

```diff
@@ -0,0 +1,174 @@
+<template>
+    <VContainer
+        p-0
+        class="users-table"
+    >
+        <VRow
+            class="header my-3"
+            fluid
+        >
+            <VCol>
+                <VSelect
+                    class="option-select"
+                    prefix="Show"
+                    :items="options"
+                    flat
+                    v-model="selected"
+                    @change="changeSelect"
+                />
+            </VCol>
+            <VCol>
+                {{ option }}
+            </VCol>
+        </VRow>
+        <VDataTable
+            class="caption"
+            hide-default-footer
+            hide-default-header
+            :headers="headers"
+            :items="items"
+        />
+    </VContainer>
+</template>
+
+<script>
+    export default {
+        name: 'GroupedTable',
+        props: {
+            items: {
+                type: Array,
+                required: true
+            },
+            option: {
```

Comment:
what `option` means?

Reply:
ran of out of ideas of how to name it, it's the parameter we want to receive i.e. page_load_time/domain_response..., but couldn't call it 'parameter' since it's reserved for the filter[parameter] and that would be confusing

Reply:
`label` or `title` maybe?